### ISSUE TITLE
MODS?

### ISSUE THREAD

**Issue by DatFIXER** [finndev/PokeBuddy#870]
Are there any mod left? XD i can see many duplicates and many posts solved still open.
If needed i can help, i'm here most of the time (just saying, i want to help those nice guys giving us this awesome software and costantly working on it).

**Comment by azydub** [comment]
Please do! Most of your suggestions worked but Niantic may have caught onto something because trying your suggestions in the previous post 'In depth issues' doesn't seem to work anymore :( 
I also want to thank the guys making/constantly developing the software, I made an account just to thank you guys lol.

**Comment by tumadreee** [comment]
Agreed. I'm on here all day. I can help remove and consolidate threads if needed.

**Comment by finndev** [comment]
Email me at support@epicdevs.de if you are interested in helping out, thanks a lot

**Comment by dzdam** [comment]
+1, just in case
